index.html

```diff
@@ -188,6 +191,28 @@ <h5>Skills</h5>
                         </div>
                     </div>
                 </section>
+                <footer>
+                    <section id="contact-form">
+                        <div class="contact-details">
+                            <h2>Contact me</h2>
+                            <p>If you have an application you are interested in developing, a feature that you need
+                                built or a project that needs coding. I’d love to help with it!</p>
+                        </div>
+                        <div class="form">
+                            <form action="https://formspree.io/f/xzbqjwon" method="post">
+                                <input type="text" name="name" placeholder="Lorem Ipsum" maxlength="30" required>
```

Comment:
- [ ] [OPTIONAL] Great implementation, partners! :+1: But since you are designing for humans, and the purpose of placeholders is to represent what the original form looks like, sort of a guide that explains the intended content. Also here, to make your users understand what the `input` is meant for clearly, I think using a more readable `placeholder text` would be more user-friendly. Say for example, `placeholder="Your name here"` or `placeholder="John Doe"`. How is that? :astonished: